services/src/pro/contexts/migrations/migration_0010_s2_stack_time_buffers.rs

```diff
@@ -0,0 +1,32 @@
+use async_trait::async_trait;
+use tokio_postgres::Transaction;
+
+use super::database_migration::{ProMigration, ProMigrationImpl};
+use crate::{contexts::Migration0010S2StacTimeBuffers, error::Result};
+
+#[async_trait]
+impl ProMigration for ProMigrationImpl<Migration0010S2StacTimeBuffers> {
+    async fn pro_migrate(&self, tx: &Transaction<'_>) -> Result<()> {
+        // add priority column to layer_providers table and move description column to the definition types
+        tx.batch_execute(
+            r#"
+            CREATE TYPE "StacQueryBuffer" AS (start_seconds bigint, end_seconds bigint);
```

Comment:
why are start and end individually configurable? Do you need this?

Reply:
i think it is helpfull when we work with different kinds of granularity. It does not hurt to be able to set them individually, right?.